src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/citizen/CCJByAdmissionPdfTest.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.hmcts.cmc.claimstore.tests.functional.citizen;
+
+import io.restassured.RestAssured;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import uk.gov.hmcts.cmc.claimstore.idam.models.User;
+import uk.gov.hmcts.cmc.claimstore.tests.functional.BaseClaimPdfTest;
+import uk.gov.hmcts.cmc.claimstore.utils.Formatting;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.domain.models.response.Response;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleCountyCourtJudgment;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponse;
+
+import java.io.IOException;
+
+import java.util.function.Supplier;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatMoney;
+
+public class CCJByAdmissionPdfTest extends BaseClaimPdfTest {
+
+    @Before
+    public void before() {
+        user = idamTestService.createCitizen();
+    }
+
+    @Test
+    public void shouldBeAbleToFindAppropriateDataInCCJByAdmissionPdf() throws IOException {
+        Claim createdCase = createCase();
```

Comment:
create case down to submitResponse should be done in the `@Before` they are setup and irrelevant to the test case